src/Core/Prefab/PanoramaView.js

```diff
@@ -57,9 +57,9 @@ PanoramaView.prototype.addLayer = function addLayer(layer) {
     if (!layer) {
         return new Promise((resolve, reject) => reject(new Error('layer is undefined')));
     }
-    if (!layer.isColorLayer) {
-        throw new Error(`Unsupported layer type ${layer.type} (PanoramaView only support 'color' layers)`);
-    }
+    // if (!(layer.isColorLayer || layer.isGeometryLayer)) {
```

Comment:
Why adding this if you are commenting it ?

Reply:
oops

Reply:
done